doc.go

```diff
@@ -23,28 +25,43 @@ func ConvertDoc(r io.Reader) (string, map[string]string, error) {
 	// Meta data
 	mc := make(chan map[string]string, 1)
 	go func() {
+		defer func() {
+			if e := recover(); e != nil {
+				log.Printf("panic when reading doc format: %v", e)
```

Comment:
`recovered panic`?

Reply:
Yeah, scanned over the metadata extraction code and there seems to be a lot of places for malformed documents to cause panics.

Just putting this here to stop it failing the doc conversion.  Noting previously we would just ignore fatal errors calling out to wvSummary, so this seemed equivalent.

Reply:
I meant change log message so we don't freak out and think it panicked when reading logs.